### ISSUE TITLE
How to track my stock portfolio

### ISSUE THREAD

**Issue by AntouanK** [maybe-finance/maybe#107]
I just cloned the repo and launched the app.
I cannot understand how to add stocks to track my portfolio.
Is that feasible? Or does it only work with US institution integrations?
( I live in the UK, I want to manually add my stocks and just track their prices for my portfolio net worth )

**Comment by Shpigford** [comment]
We don't currently have the option for adding a stock and having it track the value of it. The original app was all built around automated tracking via bank connectors (like Plaid). And at launch it was US-only.

We're presently doing the work to get the automated route re-working (via Tiller) and then we'll 100% start working on the manual stock tracking.

tl;dr Not possible for your situation just yet but 100% coming.

**Comment by AntouanK** [comment]
thanks for letting me know :+1:
